build.gradle.kts

```diff
@@ -1,4 +1,3 @@
-import net.minecrell.pluginyml.bukkit.BukkitPluginDescription
```

Comment:
I'll blame IJ on that, the import is still needed to resolve the plugin if it's not cached.